templates/census-thank-you.html

```diff
@@ -13,7 +13,7 @@
   {% call onsPanel({
     "type": "success",
     "classes": "ons-u-mb-s",
-    "icon": "check",
+    "iconType": "check",
```

Comment:
There are other uses of `icon` such as on the hub that need updating. Currently, the icons on the hub is broken. 

Jinja filters also have references to the old way which needs updating.